linetools/isgm/abssystem.py

```diff
@@ -226,6 +208,10 @@ def __init__(self, radec, zabs, vlim, zem=0., abs_type=None,
         # Kinematics
         self.kin = {}
 
+        # Metallicity
```

Comment:
The attribute in the full code is called MH, and this is ZH. I can't recall if M/H and Z/H are different quantities (this always confuses me), but I think we should be consistent in the naming.